### DISCUSSION TITLE
Lack of ability to create Pull

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [gitpython-developers/GitPython#1796] [Q&A] [answered]
Is it possible to leverage something like `gh` cli in addition to git to enable functionality to create a PR?

GitPython is great - I am doing some automation and have done everything I need to within GitPython except creating a PR! I would hate to have to shift the process because of lack of functionality there when it includes everything else I need! Thanks.

**Comment by Byron** [comment, marked as answer]
GitPython is only dealing with Git, and not with GitHub. There is [PyGitHub](https://github.com/PyGithub/PyGithub) which seems to do what you need though.

**Reply by Alex-Wenner-FHR** [reply]
I see - understood, that is too bad - I prefer GitPython to PyGitHub because of auth mechanisms that aren't supported in PyGitHub. Thanks for the update!
